### PR TITLE
add DB_PORT option to environment variables.

### PR DESCRIPTION
**Pull request by intuxicated**
https://github.com/laravel/lumen-framework/pull/94

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -7,6 +7,7 @@ APP_FALLBACK_LOCALE=en
 
 DB_CONNECTION=mysql
 DB_HOST=localhost
+DB_PORT=port
```
Comment (crynobone) [inline .env.example]: `port` is not a valid port number.

**Comment by intuxicated** [comment]
@crynobone it is now.

**Comment by lukasgeiter** [comment]
Please squash to one commit.

**Comment by intuxicated** [comment]
@lukasgeiter done.

**Comment by crynobone** [comment]
What happen when you decide to use sqlite, suddenly you **need to know** that you need to update ot remove this config.

I don't see how this is useful to be there by default.

**Comment by lukasgeiter** [comment]
Maybe you could comment it out? So people know it exists but it isn't set by default.

**Comment by intuxicated** [comment]
@crynobone You are doing it wrong ;) [For more information](https://github.com/laravel/lumen-framework/blob/5169044cb004d85c6acd017cc59eab754b9a0aec/config/database.php#L54).
Sqlite configuration is not using `DB_PORT` environment variable. As you can see, sqlite configuration has no `DB_USERNAME` or `DB_PASSWORD` environment variable either. So what do you do with them for sqlite, do it for `DB_PORT`, remove it or leave it. That's not going to hurt.

@lukasgeiter that's not a bad idea.

**Comment by crynobone** [comment]
@intuxicated what about Postgres https://github.com/laravel/lumen-framework/blob/5169044cb004d85c6acd017cc59eab754b9a0aec/config/database.php#L77

**Comment by intuxicated** [comment]
@crynobone You got to change your driver, you can change your port number too, Its not a big deal.

**Comment by lukasgeiter** [comment]
It's not a big deal but without it being set as environment variable you wouldn't have to change the port. (at least if you are using the default)

**Comment by intuxicated** [comment]
@lukasgeiter As i said before, I'm agree to make this variable optional. but @crynobone Is talking about removing this variable. which I'm not agree.

**Comment by crynobone** [comment]
> which I'm not agree.

The environment files missing `DB_PREFIX`, `DB_TIMEZONE` too, why not include those as well?

**Comment by intuxicated** [comment]
@crynobone That's good idea. I'll make another pull request or modify this if they agree with us.

**Comment by intuxicated** [comment]
>The environment files missing DB_PREFIX, DB_TIMEZONE too, why not include those as well?

@taylorotwell what you think ?

**Pull request merged**